src/apis/enableMeasurementTool.js

```diff
@@ -0,0 +1,6 @@
+import { PRIORITY_ONE } from 'constants/actionPriority';
```

Comment:
Is this file just for testing purpose?
We already have an `enableMeasurement` API so I don't think we need this API.